src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -147,27 +109,30 @@ export function ServiceCatalogItemPage({
       ? organizations[0]?.id?.toString()
       : null;
 
-  const toggleDescription = () => {
-    setIsExpanded(!isExpanded);
-  };
+  useEffect(() => {
+    async function fetchServiceCatalogItem() {
+      try {
+        const response = await fetch(
+          `/api/v2/help_center/service_catalog/items/${serviceCatalogItemId}`
+        );
+        const data = await response.json();
+        if (response.ok) {
+          setServiceCatalogItem(data.service_catalog_item);
+        }
```

Comment:
Should we move `const data = ....` to the if block?
```suggestion
        if (response.ok) {
            const data = await response.json();
          setServiceCatalogItem(data.service_catalog_item);
        }
```

Reply:
Yes we can do this, I moved it.